src/db/dbquery.cpp

```diff
@@ -251,9 +255,9 @@ std::vector<db::utils::Address> db::DBQuery::get_route_list(uint32_t p_id) {
             /*for (Json::Value::ArrayIndex i = 0; i != response.size(); i++) {
                 LOG(ltrace, response[i]["port"].asString());
             }*/
```

Comment:
This piece of commented code should be deleted

Reply:
done